assets/dev/js/frontend/handlers/video.js

```diff
@@ -133,7 +134,13 @@ export default class Video extends elementorModules.frontend.handlers.Base {
 			return;
 		}
 
-		this.apiProvider.onApiReady( ( apiObject ) => this.prepareYTVideo( apiObject ) );
+		// Due to a bug in the YouTube Embed API, there might be a conflict when privacy mode is set to ON, and the
+		// page contains additional YouTube videos that are loaded WITHOUT privacy mode, such as a video background
+		// in a section. In these cases, to avoid the conflict, a timeout is added to defer the initialization of
+		// the Youtube API object.
+		setTimeout( () => {
```

Comment:
Set timeout only when "improved assets loading" is disabled

Reply:
Done.